bookwyrm/static/js/bookwyrm.js

```diff
@@ -368,4 +374,84 @@ let BookWyrm = new class {
             );
         }
     }
+
+    handleModalButton(element) {
+        const modalButton = element.currentTarget;
+        const targetModalId = modalButton.dataset.modalOpen;
+        const htmlElement = document.querySelector('html');
+        const modal = document.getElementById(targetModalId);
+        
+        if (!modal) {
+            return;
+        }
+
+        // Helper functions
+        function handleModalOpen(modalElement) {
+            htmlElement.classList.add('is-clipped');
```

Comment:
Is this to prevent the background from scrolling?

Reply:
yes, it's recommended by the Bulma doc

> To activate the modal, just add the `is-active` modifier on the `.modal` container. You may also want to add `is-clipped` modifier to a containing element (usually html) to stop scroll overflow.